polar2cartesian.py

```diff
@@ -0,0 +1,13 @@
+from math import cos, sin
+
+# Read values for r and θ
+r = float(input("r = "))
+θ = float(input("θ = "))
+
+# Convert polar coordinates to cartesian
+x = r*cos(θ)
+y = r*sin(θ)
+
+# Print results
+print("x = " + str(x))
+print("y = " + str(y))

```

Comment:
**Suggestion:** It's a good practice to encapsulate the conversion logic into a function. This will make the code more modular, reusable, and testable.
```suggestion
def polar_to_cartesian(r, θ):
    x = r*cos(θ)
    y = r*sin(θ)
    return x, y

r = float(input("r = "))
θ = float(input("θ = "))
x, y = polar_to_cartesian(r, θ)
print("x = " + str(x))
print("y = " + str(y))
```